### DISCUSSION TITLE
The camera sees propellers.

### DISCUSSION THREAD

**Discussion by noahzn** [bitcraze/discussions#428] [Q&A] [answered]
I'm running the wifi-img-streamer, and the camera can see two front propellers. Is this expected? I haven't tested it when flying, maybe the propellers are not visible when they rotate fast? Thank you.

**Comment by knmcguire** [comment, marked as answer]
Yes! That is true, but they are hardly noticeable once you fly. You can check out the FPV video we did here where you can see it in flight: https://youtu.be/zLxYxvWMohQ
